src/Package/Impl/DataInspect/VisualGrid/MatrixViewAutomationPeer.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
```

Comment:
I am wondering if we should pull grid out and into a separate assembly like `Wpf.Controls` or something. Might come useful to PTVS. Probably not now, just saying.